models/me.js

```diff
@@ -0,0 +1,15 @@
+var mongoose = require('mongoose');
+ var Schema = mongoose.Schema;
+
+ // me schema
+ var Me = new Schema({
```

Comment:
We specifically said not to put your profile information in the database, but that seems to be the only piece you tried to set up at all.